src/formio/components/partners.ts

```diff
@@ -0,0 +1,21 @@
+import {TableViewComponentSchema} from '..';
+
+/**
+ * @group Form.io components
+ * @category Base types
+ */
+export interface PartnerDetails {
+  bsn?: string;
+  initials?: string;
+  affixes?: string;
+  lastName?: string;
+  dateOfBirth?: string;
```

Comment:
are these really optional fields that may be missing from the object? Can't we normalize this on the backend to make sure at least the keys are always provided, but possibly with an empty string as value?

Reply:
Yes, you are right because in the backend I have empty strings in case we do not have a value.